documentation/src/main/asciidoc/topics/proc_registering_sci_remote_caches.adoc

```diff
@@ -0,0 +1,55 @@
+[id='registering-sci-remote-caches_{context}']
+= Registering serialization context initializers with {brandname} Server
+
+You must register Protobuf schema with {brandname} Server if you want other clients to operate on your custom types, perform Ickle queries, or convert custom types between media types.
```

Comment:
```suggestion
You must register Protobuf schema with {brandname} Server if you want to perform Ickle queries, convert from `application/x-protostream` to other media types like `application/json`.
```
I don't think you need the schema on the server for other clients to operate on your custom types, the other clients can get the schema elsewhere (and having just the schema doesn't mean they can unmarshall your POJOs, if they don't have the classes and the generated marshallers).

Reply:
I've changed this